crates/ruff_workspace/src/configuration.rs

```diff
@@ -916,9 +917,31 @@ impl LintConfiguration {
 
                 // Deprecated rules
                 if kind.is_enable() {
-                    if let RuleSelector::Rule { prefix, .. } = selector {
+                    if let RuleSelector::Rule {
+                        prefix,
+                        redirected_from: None,
+                    } = selector
```

Comment:
Can this be a `match` instead of mutually-exclusive `if`-`let`? Or am I misreading?

Reply:
Yes I guess if we want to be explicit about the cases we are ignoring. I'm going to address this in the next commit in the stack though, if that's okay?